src/types/geoNear.ts

```diff
@@ -0,0 +1,50 @@
+
```

Comment:
These are actually already defined already.  Use those definitions instead (or update them if you don't think they're correct).

Reply:
Thanks @tianhuil  for your review comments, however, I could only reuse `Coordinates` from `geometry.ts` other types incompatible

Reply:
Now renamed to `TsLegacyCoordinates.ts` from `geojson.ts`